packages/keystone/lib/Keystone/index.js

```diff
@@ -263,17 +263,16 @@ module.exports = class Keystone {
     return graphql(schema, query, null, context, variables);
   }
 
-  // The GraphQL App uses this method to build up the context required for each
-  // incoming query.
-  // It is also used for generating the `keystone.query` method
-  getGraphQlContext({ schemaName, req = {}, skipAccessControl = false } = {}) {
+  createHTTPContext({ schemaName, req }) {
+    // The GraphQL App uses this method to build up the context required for each incoming query.
     return {
       ...this.createContext({
         schemaName,
         authentication: { item: req.user, listKey: req.authedListKey },
-        skipAccessControl,
+        skipAccessControl: false,
```

Comment:
is this going to break use when want to skip access control

Reply:
That's a default value so if you create a context with `skipAccessControl: true`, no prob.